### DISCUSSION TITLE
Bem vindos as discussões do Data Career Guide

### DISCUSSION THREAD

**Discussion by 1cadumagalhaes** [datacareer-guide/datacareer-guide.github.io#1] [Announcements]
## 👋 Bem-vinde!
  Estamos usando as Discussões como um lugar para nos conectarmos com outros membros da nossa comunidade. Esperamos que você:
  * Faça perguntas sobre as quais você está se perguntando.
  * Compartilhar ideias.
  * Envolva-se com outros membros da comunidade.
  * Dê boas-vindas aos outros e tenha a mente aberta. Lembre-se que esta é uma comunidade que
  construir juntos 💪.

  Para começar, comente abaixo com uma apresentação sua e conte-nos sobre o que você faz com esta comunidade.

**Comment by ghost** [comment]
Olá a todos.

esse projeto está ativo?

*gostaria de contribuir com algumas ideias*
